DeAceroDLabs/marketplace-frontend
# Feature/dmp 666 Notification component

## What is done

[[DMP-666] Notification component](https://deacero.atlassian.net/browse/DMP-666)

## How is done

- Created a `Notification` component that will be used when user click on Agregar al carrito.
- Created a `notificationOpenContext` so that we can open or close notification throughout all the routes.

## How to test

1. Run the project via `npm start`
2. Click on any product.
3. Click `Agregar al carrito` button.
4. This button should navigate to home with and show a notification.
5. You should see be able to close the notification with the close button on the top right.
6. You should be able to close notification by clicking outside of it.
